Project.toml

```diff
@@ -0,0 +1,16 @@
+name = "CLFFT"
+uuid = "fe1fc2a7-b730-4eae-aba2-587af0173dc9"
+authors = []
+version = "0.1.0"
+
+[deps]
+BinDeps = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
```

Comment:
Are you sure we need BinDeps.jl? I thought that BinaryProvider.jl was the new way? Aren't they mutually exclusive?

Reply:
Yes, they are.
I removed BinDeps